extensions/example-extension/renderer.tsx

```diff
@@ -30,13 +55,48 @@ export default class ExampleExtension extends LensRendererExtension {
     }
   ];
 
-  clusterPageMenus: Interface.PageMenuRegistration[] = [
+  clusterPageMenus: Interface.ClusterPageMenuRegistration[] = [
     {
+      id: "top-example-menu",
       title: "Example extension",
       components: {
         Icon: ExampleIcon,
       },
     },
+    {
+      parentId: "top-example-menu",
+      title: "Simple Page",
+      target: {
+        pageId: "simple-page"
+      },
+      components: {
+        Icon: ExampleIcon,
+      },
+    },
+    {
+      parentId: "top-example-menu",
+      title: "Simple Params",
+      target: {
+        pageId: "page-with-simple-parms",
+        params: {
+          exampleId: "no-secret",
+          namespace: "test"
+        }
+      },
+      components: {
+        Icon: ExampleIcon,
+      },
+    },
```

Comment:
this menu item doesn't show up at all (with or without `parentId`) so I may have a bug somewhere?

Reply:
I haven't define/updated any logic related to `parentId`/`id` submenus relations in my PR, so have to check how it works actually. 
I assume you must define `parentId` only when same item existing in menu items registry (and you know it for sure), @jakolehm right?

Reply:
typo in the pageId! should've used a var